docs/openapi/schemas.yml

```
@@ -1,25 +1,25 @@
 ---
 #### REQUESTS ####
 
-requests_oauth_post_login_email:
-  $ref: "schemas/requests/oauth/post_login_email.yml"
+requests_oauth_token_password:
+  $ref: "schemas/requests/oauth/token_password.yml"
 
-requests_oauth_post_refresh_token:
-  $ref: "schemas/requests/oauth/post_refresh_token.yml"
+requests_oauth_token_refresh_token:
+  $ref: "schemas/requests/oauth/token_refresh_token.yml"
 
-requests_oauth_post_revoke:
-  $ref: "schemas/requests/oauth/post_revoke.yml"
+requests_oauth_revoke:
+  $ref: "schemas/requests/oauth/revoke.yml"
 
-requests_registrations_post_create:
-  $ref: "schemas/requests/registrations/post_create.yml"
+requests_registrations:
+  $ref: "schemas/requests/registrations.yml"
 
 #### RESPONSES ####
 
-responses_oauth_post_token:
-  $ref: "schemas/responses/oauth/post_token.yml"
+responses_oauth_token:
+  $ref: "schemas/responses/oauth/token.yml"
 
-responses_surveys_get_survey_list:
-  $ref: "schemas/responses/surveys/get_survey_list.yml"
+responses_surveys:
+  $ref: "schemas/responses/surveys.yml"
```

After rechecking, I think we can add the RESTful action name here:
```suggestion
responses_surveys_index:
  $ref: "schemas/responses/surveys/index.yml"
```
Then later, we can will have another for resource detail:

```yml
responses_surveys_index:
  $ref: "schemas/responses/surveys/show.yml"
```

Same for `update`/`delete`
What do you think?

---

@hoangmirs That's a good raise. I was thinking about following the RESTful action names. However when checking with OpenAPI, it supports `get`, `post`, `put`, `patch`, `delete`, `head`, `options`, and `trace` operations. So I prefer to use operations for easy-to-find and expandable.

Moreover, for surveys show (endpoints: `surveys/{id}`), it's defined with another path, so we can do this

For examples
```yml
# openapi.yml

paths:
  /surveys:
    $ref: "paths/surveys.yml"

  /surveys/{id}:
    $ref: "paths/surveys_id.yml"
```
```yml
# paths/surveys.yml

get:
  $ref: "paths/surveys/get.yml"

post
  $ref: "paths/surveys/post.yml"
```
```yml
# paths/surveys_id.yml
get:
  ...
```